.circleci/real_config.yml

```diff
@@ -4403,6 +4403,28 @@ workflows:
           mark: port_registry
           target-stage: agent
 
+      - test-e2e:
```

Comment:
Please do not add two entire new series of e2e tests.  They are expensive and slow.  What kinds of unit tests could you write that would help cover this feature?

What kinds of unit tests can we write to make sure this feature is working properly?

Reply:
e2e tests removed and unit tests added